src/effect.cpp

```diff
@@ -317,6 +317,14 @@ bool effect_type::load_mod_data(JsonObject &jsobj, std::string member) {
         extract_effect(j, mod_data, "vomit_chance_bot", member, "VOMIT",    "chance_bot");
         extract_effect(j, mod_data, "vomit_tick",       member, "VOMIT",    "tick");
 
+        // Then healing effects
+        extract_effect( j, mod_data, "healing_rate",                 member, "HEAL_MED_RATE", "amount" );
+        extract_effect( j, mod_data, "healing_rate_w_other_effects", member, "HEAL_MED_RATE_W_OTHER", "amount" );
+        extract_effect( j, mod_data, "healing_mult_regen",           member, "HEAL_MED_REGEN", "amount" );
+        extract_effect( j, mod_data, "healing_head_mult", member, "HEAL_HEAD_MULT", "amount" );
```

Comment:
Indent these as well.

Reply:
It does not look much better after Astyle with `--style=1tbs --attach-inlines --indent=spaces=4 --align-pointer=name --max-code-length=100 --break-after-logical --indent-classes --indent-preprocessor --indent-switches --indent-col1-comments --min-conditional-indent=0 --pad-oper --unpad-paren --pad-paren-in --add-brackets --convert-tabs`.

Reply:
`effect.cpp` is in `astyle_blacklist`. It can violate astyle if needed.